### DISCUSSION TITLE
Orchid install with example for User management

### DISCUSSION THREAD

**Discussion by dhb52** [orchidsoftware/platform#1522] [Show and tell]
Following the official guide for install orchid, there's no user management out of box.
While there's `Access management` on the user drop-down menu.
User creation function lacks.

While the official document now is not so sophisticated, please add some code example

**Comment by tabuna** [comment]
I didn't add user creation out of the box because you usually need different ways:

- Simple registration. To solve this, I created a package https://github.com/orchidsoftware/fortify
- Inviting the user. The first administrator has the ability to invite a user (by email) with pre-set permissions/role where the user himself enters his name, password, and other data.
 - Administrator adds a user manually, indicating all user data.

Which of these (or other) ways would you like to see? Do you want to work on it?

**Comment by dhb52** [comment]
Thanks for the quick reply, I came across this project a few days ago.

Actually, I suggested adding a few code to get `user/role/permission management` into a system menu as example code,
which may help get familiar with this project more quickly.

btw, user creation way I meant `Administrator adds a user manually, indicating all user data.`
